hs_core/migrations/0055_auto_20201031_1727.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-10-31 17:27
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def remove_comma_from_creator_contributor_for_empty_author_name(apps, schema_editor):
+    Creator = apps.get_model('hs_core', 'Creator')
+    for creator in Creator.objects.all():
+        if creator.name == ', ':
+            creator.name = ''
+            creator.save()
+    Contributor = apps.get_model('hs_core', 'Contributor')
+    for contributor in Contributor.objects.all():
+        if contributor.name == ',' or contributor.name == ', ':
+            contributor.name = ''
+            contributor.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hs_core', '0054_auto_20201028_1432'),
+    ]
+
+    operations = [
+        migrations.RunPython(remove_comma_from_creator_contributor_for_empty_author_name),
+    ]
```

Comment:
consider a meaningful name instead of auto for the migration

Reply:
this can get annoying and the majority are using the auto convention, so understood if you take no action

Reply:
auto migration file is automatically generated by Django, so I don't think I should change it.